test/e2e/dashboard.go

```diff
@@ -28,28 +28,28 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("kube-ui", func() {
+var _ = Describe("Kubernetes Dashboard", func() {
 	const (
-		uiServiceName = "kube-ui"
+		uiServiceName = "kubernetes-dashboard"
 		uiAppName     = uiServiceName
 		uiNamespace   = api.NamespaceSystem
 
 		serverStartTimeout = 1 * time.Minute
 	)
 
-	f := NewFramework("kube-ui")
+	f := NewFramework(uiServiceName)
 
-	It("should check that the kube-ui instance is alive [Conformance]", func() {
-		By("Checking the kube-ui service exists.")
+	It("should check that the kubernetes-dashboard instance is alive [Conformance]", func() {
```

Comment:
/cc @ihmccreery -- how do we feel about changing conformance tests in this way? This means that the conformance test from the 1.1 branch (checking for kube-ui) will fail against 1.2 clusters and the conformance test from the 1.2 branch (checking for the dashboard) will fail against 1.1 clusters.